### ISSUE TITLE
Howto:howto

### ISSUE THREAD

**Issue by jirmueller** [joaompneves/HyperlapseBatchProcessor#1]
Can you please explain how to build this tool?
I tried with c# but it trows errors like "HyperlapseWrapper.cs is not in directory" - of course it is.
Thanks a bunch

**Comment by joaompneves** [comment]
Have you tried opening the sln on Visual Studio and building it?

**Comment by fabceolin** [comment]
When I tried to build the project, there are a lot undefined calls, like: CalibrationProvider, CalibrationMatcher, Rational, etc..

![screenclip](https://cloud.githubusercontent.com/assets/647489/13728409/f9bf6ba6-e8f5-11e5-85ca-e8845ed015f5.png)

**Comment by joaompneves** [comment]
This project was developed using a previous release of the MS Hyperlapse. They have probably changed the APIs in recent versions. Right now I'm not maintaining this project, neither have the time to do it. Feel free to fork it, I'll accept PRs.

**Comment by ErfolgreichCharismatisch** [comment]
Yes, they changed it. Currently I am missing VideoReader and Microsoft.Research.Hyperlapse.Desktop and CalibrationMatcher  CalibrationProvider VideoBitrateEstimator, HyperlapseEngine HyperlapseParameters and ProcessingFailedEventArgs

Please maintain the project again or tell us, where you got your information from on how to use it

**Comment by koronamou** [comment]
Hi!

Very interesting project! Bravo!

I tried with v1.6 Hyperlapse and I have managed to set the class given, using correct

using Microsoft.Research.Hyperlapse;
using Microsoft.Research.Hyperlapse.Desktop;
using Microsoft.Research.VisionTools.Toolkit;
using Microsoft.Research.VisionTools.Toolkit.Desktop.Native;

But I am having problem to build the project, due error in line 23:
var engine = new **HyperlapseEngine**();

It is missing arguments like **INativeHyperlapseEngine** and **ProductInfo**

Could you advise please?

I know it is different version from that you have build, but I need a little help on this one HyperLapse v1.6

p.s.

With previous version of Hyperlapse when I set UseAdvancedSmoothing= true I am getting log files as an output. From what I understand it seems that it needs more parameterization. Do you have any Idea of how to set it correct on Advancemode?

Thanks in advance
